src/DotVVM.Samples.Common/ViewModels/ControlSamples/DataPager/DataPagerViewModel.cs

```diff
@@ -11,7 +11,7 @@ public class DataPagerViewModel : DotvvmViewModelBase
         public GridViewDataSet<Data> DataSet { get; set; }
         public override Task Init()
         {
-            DataSet = GridViewDataSet.Create(GetData, pageSize: 3);
```

Comment:
Do not delete the previous working use case, it should work too. We need to cover both options with the test.